cmd/root.go

```diff
@@ -126,9 +139,20 @@ func initGlobalConfig() {
 	cmdconfig.SetViperDefaults(steampipeconfig.GlobalConfig.ConfigMap())
 
 	// now validate all config values have appropriate values
-	if err := validateConfig(); err != nil {
-		panic(err)
-	}
+	err = validateConfig()
+	utils.FailOnErrorWithMessage(err, "failed to validate config")
+}
+
+// migrate all data files to use snake casing for property names
+func migrateLegacyFiles() error {
+	return utils.CombineErrors(
```

Comment:
```
// skip migration form plugin manager because....
if viper.Get(constants.ConfigKeyActiveCommand).(*cobra.Command).Name() == "plugin-manager"{
  return nil
}